openedx/core/djangoapps/course_groups/views.py

```diff
@@ -502,6 +502,9 @@ class CohortHandler(DeveloperErrorViewMixin, APIPermissions):
             * user_partition_id: The integer identified of the UserPartition.
             * group_id: The integer identified of the specific group in the partition.
     """
+    def get_queryset(self):
```

Comment:
Try changing the parent's `GenericAPIView` to `APIView` and run tests locally to confirm if it does not break anything

Reply:
I think it is better to add a queryset attribute rather override this method